performance-tests/pom.xml

```diff
@@ -104,6 +104,12 @@
         <plugins>
 
             <plugin>
+                <groupId>org.jacoco</groupId>
+                <artifactId>jacoco-maven-plugin</artifactId>
+                <version>${jacoco.plugin.version}</version>
+            </plugin>
```

Comment:
Coverage *must* not be on for performance tests or jmh tests. Jacoco shouldn't even be on the classpath. As long as its off for some modules, I think it's desirable to turn it off for most modules. Or rather, only turn it on in the two modules where it's relevant, unit-tests and unit-tests-java8.